File: app/src/main/java/com/starostinvlad/fan/GsonModels/Datum.java
package com.starostinvlad.fan.GsonModels;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

import androidx.room.Embedded;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Datum implements Serializable {

    @PrimaryKey(autoGenerate = true)
    public long id;

    @SerializedName("serial")
    @Expose
    @Embedded(prefix = "serial")
    private Serial serial;
    @SerializedName("episode")
    @Expose
    @Embedded(prefix = "episode")
    private Episode episode;

    public Serial getSerial() {
        return serial;
    }

    public void setSerial(Serial serial) {
        this.serial = serial;
    }

    public Episode getEpisode() {
        return episode;
    }

    public void setEpisode(Episode episode) {
        this.episode = episode;
    }

}

File: app/src/main/java/com/starostinvlad/fan/GsonModels/Datum.kt
package com.starostinvlad.fan.GsonModels

import androidx.room.Embedded
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable
import kotlin.Throws

@Entity
class Datum : Serializable {
    @kotlin.jvm.JvmField
    @PrimaryKey(autoGenerate = true)
    var id: Long = 0

    @SerializedName("serial")
    @Expose
    @Embedded(prefix = "serial")
    var serial: Serial? = null

    @SerializedName("episode")
    @Expose
    @Embedded(prefix = "episode")
    var episode: Episode? = null
}